HW7/IhorBohush/07-1_Practical_task_2_Bohush.py

```diff
@@ -0,0 +1,47 @@
+# Write a program that calculates the area of a rectangle,
+# triangle and circle
+# (write three functions to calculate the area, and call them in the
+# main program depending on the user's choice).
+from math import pi
+
+
+def rectangle():
+    length = float(input('Enter the length of the rectangle: '))
+    width = float(input('Enter the width of the rectangle: '))
+    area_rectangle = length * width
+    return area_rectangle
```

Comment:
As mentioned above: when a variable is created only to be returned immediately, we can skip creating the variable and return the statement directly